### ISSUE TITLE
TE spacing changes with ME-EPI after upgrade to PrismaFit/XA30/R017

### ISSUE THREAD

**Issue by krevill** [CMRR-C2P/MB#302]
Our site recently upgraded from a Trio (VB11) to a PrismaFit (XA30), which has also necessitated the change from R016 to R017.  Most of the protocols using the CMRR MB BOLD sequences have transferred without issue, including some using MB/ME with 2 or 3 'echoes'. However, we are having trouble replicating the TE spacing for a protocol using 2MB/5ME. The lowest TE is the same but the upper ones get increasingly farther away from where they were before, and the research group is unhappy about increased dropout at the longer TEs relative to before. We also had to reduce slices to keep the same TE.  I have verified that inter-TE delay is set to 0.  I've had a few of our local physicists take a look but they aren't finding anything to tweak that fixes the issue.     I can attach before/after parameter PDFs if helpful. Any suggestions?

**Comment by eauerbach** [comment]
Trio to PrismaFit is a pretty significant change independent of the software version. Can you make sure you are using Gradient mode = Performance, and you will likely need to re-optimize the bandwidth to minimize the echo spacing (which will reduce the minimum TE and also reduce the TE spacing).

**Comment by dfabbott** [comment]
We’ve also recently upgraded scanners. We’re unable to match BOLD scan parameters of R2016a VE11C on Skyra compared to R017pre6 XA50 on Vida (Performance Gradient mode). We cannot achieve as low a TR or TEs. With common parameters:
FoV read= 216 mm
Slice thickness= 3.00 mm
Multi-band accel. factor 4
PAT mode GRAPPA
Accel. factor PE 2
Multi-slice mode Interleaved
Series Interleaved
Multi-band accel. factor 4

On the Skyra we achieved, with considerable headroom: 
TR= 900 ms
TE1= 15.00 ms
TE2= 34.37 ms
TE3= 53.74 ms

Whereas on VIDA we can only achieve:
TR= 912.0 ms
TE1= 15.00 ms
TE2= 35.14 ms
TE3= 55.26 ms

@eauerbach, I can send to you the full sequence cards for each scanner for comparison if that helps.

**Comment by eauerbach** [comment]
Check the Echo Spacing on the Sequence / Part 1 card. On Skyra, you were running at an echo spacing of 0.52 ms, but it is 0.54 ms on Vida. Echo spacing of 0.52 ms is a switching frequency of 1/(2*.52 ms) = 962 Hz. Unfortunately one of the forbidden frequency bands on Vida is 940-1240 Hz (these are forbidden mainly due to acoustic resonances of the particular magnet/gradient system). On Skyra the similar restricted band is 1010-1230 Hz. So on Vida the system forces you to drop down to the closest permissible echo spacing of 0.54 ms or 926 Hz. This is the difference.

**Comment by dcbrien** [comment]
I thought I would just add to this as we also upgraded from Trio to Prisma a few years back.

I noticed recently that the backup and restore during the upgrade copied all of the old Trio wip .dlls over to the mricustomer/seq directory. They didn't seem to interfere with the new .exe installer for VE11E and we have had no issues, so I assume it just ignores them.

Another thing I noticed is that when we tried to import a .exar from a VE11C build to our VE11E, we couldn't quite achieve the exact acceleration parameters for a multi-echo sequence. On VE11C they had pF 6/8 and GRAPPA at 32 lines, but I could only enable 24 lines, which slightly changed the TEs. We think it might be an order of parameter selection thing, but I haven't been able to 100% replicate still. Also, in the underlined upgrade notes of the sequence, it has a line that says "!! ARRAY NAMETAG seq_wip4 !!!: 9 -> 8" and I'm not sure how to interpret that.
